schemas/com.snowplowanalytics.snowplow.media/ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad_break",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Ad break name (e.g., pre-roll, mid-roll, and post-roll).",
```

Comment:
Nit: Trailing period in descriptions seems inconsistent between the various schemas.

Reply:
Unified on adding the period to all descriptions.